internal/datastore/postgres/postgres_test.go

```diff
@@ -210,7 +219,7 @@ func GarbageCollectionTest(t *testing.T, ds datastore.Datastore) {
 	require.NoError(err)
 	require.Zero(removed.Relationships)
 	require.Equal(int64(1), removed.Transactions) // firstWrite
-	require.Zero(removed.Namespaces)
+	require.Equal(int64(2), removed.Namespaces)   // resource, user
```

Comment:
why did this change?

Reply:
It actually changed back, and it's because we no longer need to keep around the stale transaction. This is essentially the reverse of this change:
https://github.com/authzed/spicedb/commit/fbaee5b00b34116ed1a14e6e0f8c27b2491d76cd